cmd/agent/install_script.sh

```diff
@@ -567,58 +602,58 @@ Please follow the instructions on the Agent setup page:
     exit;
 fi
 
-if [ "$upgrade" ]; then
+if [ "$upgrade" ] && [ "$agent_flavor" != "datadog-dogstatsd" ]; then
```

Comment:
nit: I would check if agent flavor is datadog-agent or datadog-iot-agent because of the same reason as above (but I think it is less important here)

Reply:
Well, I think this is twofold. If we add that check and then add another "agent-like" flavor (like IoT Agent), we would have to extend that condition to cover that as well. At this point, I think it's fair to assume that a new flavor we would add would be "agent-like" - so I think having the condition as-is is actually less error prone for future.

Reply:
That's fair, let's keep it like this then